survey/migrations/0002_auto__add_field_question_qtext_en__add_field_question_qtext_es__add_fi.py

```diff
@@ -115,6 +187,7 @@ def backwards(self, orm):
         u'survey.subjectid': {
             'Meta': {'ordering': "['subjectid']", 'object_name': 'SubjectID'},
             u'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
+            'language': ('django.db.models.fields.CharField', [], {'default': "'en'", 'max_length': '20'}),
             'subjectid': ('django.db.models.fields.CharField', [], {'unique': 'True', 'max_length': '5'}),
             'user': ('django.db.models.fields.related.OneToOneField', [], {'to': u"orm['auth.User']", 'unique': 'True'})
         },
```

Comment:
Might be better just not to commit the migrations at all, given that we will have different ones for local machines and the server that already has it installed.